core/native/src/main/scala/cats/effect/unsafe/KqueueSystem.scala

```diff
@@ -71,9 +71,17 @@ object KqueueSystem extends PollingSystem {
 
   def metrics(poller: Poller): PollerMetrics = PollerMetrics.noop
 
+  trait Kqueue extends FileDescriptorPoller {
+    def awaitEvent(
+        ident: Int,
+        filter: Short,
+        flags: CUnsignedShort,
+        fflags: CUnsignedInt): IO[Long]
```

Comment:
Maybe this is simpler for the UX.

```suggestion
        flags: Short,
        fflags: Int): IO[Long]
```